x/ibc_query/keeper/msg_server.go

```diff
@@ -109,14 +110,15 @@ func (k Keeper) SubmitPruneCrossChainQueryResult(goCtx context.Context, msg *typ
 	if !found {
 		return nil, sdkerrors.ErrNotFound
 	}
+	name := k.GenerateQueryCapabilityIdentifier(queryResult.Id, queryResult.Sender)
 
 	queryCapID := k.GenerateQueryCapabilityIdentifier(msg.Id, msg.Sender)
```

Comment:
Shouldn't queryResult.Id and msg.Id be the same? why did you seperate name and queryCapID..?

Reply:
Yes. queryResult.Id and msg.Id is the same. but name and queryCapID is different.
`name` is created based on value stored local,  `queryCapID` is created based on requester value. This is to prevent the user other than the user who requested the cross-chain query from deleting the query

Reply:
I understand.
I think it would be better if variables' names were renamed and **leave comments**.

How about this?
name -> crossChainQueryCapId
queryCapID -> retrieveCapId